xrviz/control.py

```diff
@@ -59,9 +75,14 @@ def set_coords(self, data):
         self.describer.set_coords(self.data, var)
         self.fields.set_coords(self.data, var)
 
+    def check_is_projectable(self, *args):
+        self.projection.is_geo.disabled = not self.fields.kwargs['are_var_coords']
```

Comment:
(should really be a method done within Projection - or otherwise a "signal" on Projection - generally I would dissuade against setting inner widgets' values directly)